src/routes-data/sidebar-routes.jsx

```diff
@@ -0,0 +1,222 @@
+            {
+                path: 'application',
+                element: <div>This is a child of random</div>,
+                label: 'Application',
+            },
+        ],
+    },
+];
+export const resources = [
+    {
+        path: 'resources',
+        label: 'Resources',
+        children: [
+            {
+                path: 'api-guide',
+                element: <ApiGuide />,
+                label: 'Api Guide',
+            },
+            {
+                path: 'faq',
+                element: <Faq />,
+                label: 'FAQ',
+            },
+            {
+                path: 'json-schemas',
+                element: <Json />,
+                label: 'JSON',
+            },
+            {
+                path: 'bug-bounty',
+                element: <BugBounty />,
+                label: 'Bug Bounty',
+            },
+        ],
+    },
+];
+
+export const rest = [
+    {
+        path: '/docs',
+        element: <Quickstart />,
+        label: 'Quickstart',
+    },
+];
+// Stitching all the arrays together to one route object for the Router.
+export const sidebar_routes = [...getting_started, ...what_can_you_do, ...resources];
+
+const ImplementDropdown = route => {
+    const location = useLocation();
+    const split_current_location = location.pathname.split('/');
+    const [isActive, setIsActive] = useState(false);
+    useEffect(() => {
+        if (route.route.path) {
+            const dropdown = route.route.path;
+            const split_current_location = location.pathname.split('/');
+            if (split_current_location.includes(dropdown)) {
+                // set the dropdown useState active.
+                setIsActive(!isActive);
+            }
+        }
+    }, []);
+    const handleToggleDropdown = () => {
+        setIsActive(!isActive);
+    };
+    return (
+        <div className={styles.dropdown}>
+            <div className={`${styles.dropdownBtn} ${isActive ? styles.boldText : ''}`} onClick={handleToggleDropdown}>
+                Implement Now
+                <span className={`${styles.arrow} ${isActive ? styles.down : ''}`} />
+            </div>
+            {isActive && (
+                <div className={styles.dropdownList}>
+                    {implement_now.map(items => (
+                        <Link
+                            key={items.path}
+                            to={'getting-started/implement-now/' + items.path}
+                            className={`${styles.dropdownContent} ${
+                                items.path === split_current_location[split_current_location.length - 1]
+                                    ? styles.selected
+                                    : ''
+                            }`}
+                        >
+                            {items.label}
+                        </Link>
+                    ))}
+                </div>
+            )}
+        </div>
+    );
+};
+
+function LinkComponent({ route, path }) {
```

Comment:
do we have typings for the props?
h: not atm
a; ok fine because the router props are well understood in userland of r router